product_set/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,11 @@
+A **product set** is a list of products which end customers aren't aware, this
+list is defined by sales manager.
+
+This module aims to help salesman to quickly create several sale order lines
```

Comment:
Adapt this description for being more general.

Reply:
Changed, but every improvement highly welcome.